README.md

```diff
@@ -232,30 +258,35 @@ More information on how SBAT works can be found [here](https://github.com/rhboot
 [your text here]
 
 *******************************************************************************
-### If your SHIM launches any other components, please provide further details on what is launched.
+### If your shim launches any other components apart from your bootloader, please provide further details on what is launched.
+Hint: The most common case here will be a firmware updater like fwupd.
 *******************************************************************************
 [your text here]
 
 *******************************************************************************
-### If your GRUB2 launches any other binaries that are not the Linux kernel in SecureBoot mode, please provide further details on what is launched and how it enforces Secureboot lockdown.
+### If your GRUB2 launches any other binaries that are not the Linux kernel in SecureBoot mode, please provide further details on what is launched and how it enforces SecureBoot lockdown.
+Skip this, if you're using a different second-stage bootloader.
 *******************************************************************************
 [your text here]
 
 *******************************************************************************
 ### How do the launched components prevent execution of unauthenticated code?
+Summarize in one or two sentences, how your secure bootchain works on higher level.
 *******************************************************************************
 [your text here]
 
 *******************************************************************************
-### Does your SHIM load any loaders that support loading unsigned kernels (e.g. GRUB2)?
+### Does your shim load any loaders that support loading unsigned kernels (e.g. certain GRUB2 configurations)?
 *******************************************************************************
 [your text here]
+
 *******************************************************************************
 ### What kernel are you using? Which patches does it includes to enforce Secure Boot?
+Hint: if you ship a recent upstream kernel release, just mention that upstream contains these patches.
```

Comment:
Generally a vanilla upstream kernel only suffices if they are build with at least `LOCK_DOWN_KERNEL_FORCE_INTEGRITY`  enabled. Otherwise you will need some lockdown patches e.g.: https://salsa.debian.org/kernel-team/linux/-/tree/master/debian/patches/features/all/lockdown

Reply:
When I was writing that initial hint, I wanted to make sure people won't get overwhelmed by thinking they need to write a thesis on the exact code that is used. I think maybe we could provide examples like suggested, and some more for certain distro families or upstream releases, but maybe in a separate document, and then link to it here? Something similar to the SBAT.example.md document.